### ISSUE TITLE
Ability to conference call / merge multiple calls

### ISSUE THREAD

**Issue by dtcooper** [64characters/Telephone#603]
Hi there. No pressure, but couldn't find an issue representing this feature so thought I would add this. Thanks for the great product!

**Comment by dtcooper** [comment]
Note this is called "multi-party calling" in some implementations.

**Comment by ASeccoW** [comment]
This would be great, it's the only feature that is missing from the product.

**Comment by B-UMA** [comment]
I completely agree with you. This feature is now a must-have because it's required by home-based locations for working . So please, please, do it!
I paid for Pro version and I think this could be a Pro feature.

**Comment by dtcooper** [comment]
I don't know that I would use @B-UMA's tone exactly, but yeah, @eofster I think several folks would be willing use to upgrade to Pro for this feature. 😜

**Comment by jmaass** [comment]
I support this feature request, conferencing would be awesome. Great product besides that!

**Comment by dtcooper** [comment]
@eofster any thoughts on this one?

**Comment by renehamburg** [comment]
It would be really fantastic if there was a way to start a ConfCall. I would love to pay for this feature as an add-on.
